metadata/ComputerVision__7__metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+  "changeLog": "Release v2023.10.1",
+  "cpu": 0,
+  "description": "Backend server for the AI Computer Vision solution that detects User Interface Elements from a provided Application screenshot",
+  "displayName": "ComputerVision",
+  "gpu": 1,
+  "inputDescription": "Please use the Computer Vision Activities included in the UIAutomation package. For more information please visit the official documentation. https://docs.uipath.com/activities/other/latest/ui-automation/computer-vision-activities ",
+  "inputType": "JSON",
+  "memory": 0,
+  "mlPackageLanguage": "PYTHON36_CV",
+  "name": "ComputerVision",
+  "outputDescription": "Please use the Computer Vision Activities included in the UIAutomation package. For more information please visit the official documentation. https://docs.uipath.com/activities/other/latest/ui-automation/computer-vision-activities ",
+  "processorType": "GPU",
+  "projectId": "[project-id]",
+  "retrainable": false,
+  "stagingUri": "[staging-uri]",
+  "projectName": "UiPath Computer Vision",
+  "projectDescription": "Backend server for the AI Computer Vision solution that detects User Interface Elements from a provided Application screenshot",
+  "tenantName": "UiPath",
+  "minAIFabricVersion": "22.10",
+  "languageVersion": 3,
+  "version": 5,
+  "customVersion": "23.10.1",
+  "imagePath": "cv2310:23.10.5.2",
+  "parametersFileJSON":  "{\"probes\": [{\"probe\": \"startup\",\"initialDelaySeconds\": \"0\",\"periodSeconds\": \"1\",\"timeoutSeconds\": \"1\",\"successThreshold\": \"1\",\"failureThreshold\": \"200\"}]}"
```

Comment:
Isn't `initialDelaySeconds` too small? What is the average time of initializing the model?

Reply:
The intention behind these values is: "allow the startup probe to pass as soon as possible", so the timeout is the minimum (1 second) and there are enough retries to account for the maximum loading time on a GPU (and on a CPU for internal testing when Automation Suite deployments with a GPU won't work).

On my dev laptop I get these times (basically less than 20 seconds):

```
(CV_build) PS D:\src\ML\computer_vision\release> docker run --rm -it -p 5000:5000 --gpus all -t cv-release-test-mod licenseagreement=accept env=cv
2023-11-22 13:58:41,932 - root - INFO - Initializing: service_internal.init
2023-11-22 13:58:41,932 - root - INFO - Initializing CV service...
2023-11-22 13:58:41,932 - root - INFO - CV server version 23.10.5.2
2023-11-22 13:58:41,932 - root - INFO - Loading models
2023-11-22 13:58:42,229 - root - INFO - Cuda available: True
2023-11-22 13:58:42,229 - root - INFO - GPU Device: cuda
2023-11-22 13:58:42,229 - root - INFO - Loading TorchScript model
2023-11-22 13:58:45,300 - root - INFO - Models loaded
2023-11-22 13:58:45,300 - root - INFO - Successfully initialized CV service!
2023-11-22 13:58:45,300 - root - INFO - Initialization complete!
2023-11-22 13:58:45,300 - root - INFO - Warming up CV model...
2023-11-22 13:58:52,681 - root - INFO - Finished warming up in 7.380836301001182 seconds
2023-11-22 13:58:52,682 - root - INFO - Individual prediction times: [('img:2', 'rep:1', 3.4269647279998026), ('img:2', 'rep:2', 1.9422967820009944), ('img:2', 'rep:3', 2.011309681000057)]
2023-11-22 13:58:52,682 - root - INFO - Initialization complete
2023-11-22 13:58:52,682 - root - INFO - Serving: http://localhost:5000/
2023-11-22 13:58:52,688 - root - INFO - Successfully loaded the module
2023-11-22 13:58:52,688 - root - INFO - Finished startup sequence
```

Reply:
Got it. I missed the 200 retries part.